docs/en/Getting-Started-on-Cluster.md

```diff
@@ -155,30 +155,29 @@ alluxio.underfs.address=s3a://S3_BUCKET/S3_DIRECTORY
 Note that in the previous `Configuration for AWS` section, the required AWS credential is already setup on
 all the Alluxio nodes. This ensures those nodes have the access to this S3 bucket.
 
-If you want to enable more advanced feature for S3 UFS, please refer to this [documentation](Configuring-Alluxio-with-S3.html).
+If you want to enable more advanced features for S3 UFS, please refer to this [documentation](Configuring-Alluxio-with-S3.html).
 
 ## Start Alluxio
 
 Next, we will format Alluxio in preparation for starting Alluxio. The following command will format
-the Alluxio journal and the worker storage directory in preparation for the master and worker to
-start.
+the Alluxio journal and the worker storage directory.
```

Comment:
we don't use it to format the worker storage directory any more in this doc

Reply:
good catch, done, thanks